### PR TITLE
Update dependencies and use express.json()

### PR DESCRIPTION
**Pull request by asonni**
Update dependencies and use express.json() middleware instead of body-parser

### CODE REVIEW

**Review by yashShelatkar — APPROVED** [review]
Looks good.

File: server.js [hunk 1]
```diff
@@ -1,25 +1,29 @@
...
-const bodyParser = require("body-parser");
-const cors = require("cors");
+const express = require('express');
+const cors = require('cors');
 
-const { handleSignin } = require("./controllers/signin");
-const { handleSignout } = require("./controllers/signout");
-const { requireAuth } = require("./controllers/authorization");
-const { getLayout } = require("./controllers/drumLayout");
+const { handleSignin } = require('./controllers/signin');
```
Comment (rgavinc) [inline server.js]: - [ ] Can you change all of the single quotes back to double quotes? The project standard is to use double quotes before single quotes :)

**Review by rgavinc — CHANGES_REQUESTED** [review]
@asonni  It's looking good! Can you resolve the request I made below and also resolve the merge conflict? Thanks!

File: server.js (hunk 1 above)
Comment (asonni) [reply server.js]: @rgavinc Fixed.

**Review by rgavinc — APPROVED** [review]
Good Work!

**Pull request merged**